### PR TITLE
Minor tweaks & replacing mutate_each_ with mutate_at

### PR DESCRIPTION
**Pull request by mbjoseph**
Here you go @lwasser - looks very good. Nice to see all of this material in one place!

Most of the changes are super minor. The only substantial change that I made was replacing the deprecated `mutate_each_()` with the luxurious `mutate_at()` function.

### CODE REVIEW

**Comment by lwasser** [comment]
Oh! is that deprecated? I found a post by mr wickam himself BUT ofcourse it was probably years old. I'll check that out. THANK YOU for the feedback and i look forward to reading all of the great edits and integrating this @mbjoseph :)

File: _posts/course-materials/earth-analytics/week-10/in-class/2016-12-06-api03-rcurl-gapminder.Rmd [hunk 1]
```diff
@@ -143,6 +143,8 @@ ggplot(summary_life_exp,
 We can also create a more advanced plot - overlaying the data points on top of
 our box plot. See the <a href="http://docs.ggplot2.org" target="_blank"> ggplot documentation</a> to learn more advanced `ggplot()` plotting approaches.
 
+
+<!-- Looks like the outliers are plotted twice here -->
```
Comment (lwasser) [inline _posts/course-materials/earth-analytics/week-10/in-class/2016-12-06-api03-rcurl-gapminder.Rmd]: hey @mbjoseph -- i stole this code from Carson. Can you please clarify where you think it is plotted twice to help me adjust?

**Comment by mbjoseph** [comment]
It's not deprecated yet but is slated to be according to the docs in dplyr 0.5.0!

File: _posts/course-materials/earth-analytics/week-10/in-class/2016-12-06-api03-rcurl-gapminder.Rmd (hunk 1 above)
Comment (mbjoseph) [inline _posts/course-materials/earth-analytics/week-10/in-class/2016-12-06-api03-rcurl-gapminder.Rmd]: The outliers are shown in hot pink as part of the boxplot and shown again as part of geom_point

Comment (mbjoseph) [inline _posts/course-materials/earth-analytics/week-10/in-class/2016-12-06-api03-rcurl-gapminder.Rmd]: I would just not plot the outliers with the boxplot for a quick fix

**Comment by lwasser** [comment]
Ok – well I’m going to test it out now and will see how it works! it was weird that they had mutate_each and mutate_each_ maybe mutate_all and mutate_at help clarify the silly _ designation which isn’t very expressive now is it?

**Comment by mbjoseph** [comment]
Ah yes - if you're using those functions in a package you typically want the version with the _ at the end. Otherwise for scripts you don't!

**Comment by lwasser** [comment]
Interesting. I didn’t know that. It seemed it didn’t work on a vector of column names without the _ but then again I was learning on the fly and it got late so who knows whether I was processing info correctly. Mutate_at is much cleaner! Thank you!! I’m working through the comments now – it all looks great.

**Pull request merged**